fdbkubernetesmonitor/monitor.go

```diff
@@ -460,6 +483,25 @@ func (monitor *Monitor) Run() {
 	}(watcher)
 	go func() { monitor.WatchConfiguration(watcher) }()
 
+	// The cluster file will be created and managed by the fdbserver processes, so we have to wait until the fdbserver
+	// processes have been started. Except for the initial cluster creation his file should be present as soon as the
```

Comment:
```suggestion
	// processes have been started. Except for the initial cluster creation this file should be present as soon as the
```

Reply:
I'm doing that change in the new PR for the retry mechanism.